website/content/en/docs/kubebuilder-migration/project_migration_guide.md

```diff
@@ -0,0 +1,275 @@
+# Project Migration - Single API version operator
```

Comment:
IHMO the title is not leading to the correct info.
WDYT about something as `Migration guide for the new layout`?

Reply:
Since we are not explaining anything about `Multi-Version API` (not anytime soon I think), I've specifically mentioned Single API version operator. Will modify to `Overview - Single API version operator` if that looks fine.